recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
+  # noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   entry_points:
```

Comment:
The latest version is `0.5.7`. Could you please update to the latest? And add `setuptools` and `wheel` in `host`?

Reply:
Will do. Thanks! Can you share your logic behind adding `setuptools` and `wheel`, so that I can make sure to do this on future builds when necessary?

Reply:
They are used implicitly but for many python packages we explicitly list them in host

Reply:
done